jxl/src/color/tf.rs

```diff
@@ -171,41 +179,106 @@ pub fn srgb_to_linear_simd<D: SimdDescriptor>(d: D, samples: &mut [f32]) {
 }
 
 /// Converts the linear samples with the BT.709 transfer curve.
+// Rational polynomial approximation of 1.099 * x^0.45 - 0.099 on sqrt(x).
+// Max error ~3e-7
 pub fn linear_to_bt709(samples: &mut [f32]) {
```

Comment:
I believe this is only used in tests now, no? Can we move it to the test module and use the "naive" implementation?